slack-lint/src/main/java/slack/lint/DeprecatedAnnotationDetector.kt

```diff
@@ -16,13 +16,19 @@
 package slack.lint
 
 import com.android.tools.lint.client.api.LintClient
-import com.android.tools.lint.detector.api.Category
-import com.android.tools.lint.detector.api.Implementation
-import com.android.tools.lint.detector.api.Issue
-import com.android.tools.lint.detector.api.Severity
+import com.android.tools.lint.detector.api.*
```

Comment:
ಠ_ಠ

Reply:
For some reason the IDE didn't pick up the style. Will go through.